vassal-app/src/main/java/VASSAL/configure/ListKeyCommandsDialog.java

```diff
@@ -297,7 +354,7 @@ else if (target instanceof GamePiece) {
               }
             }
 
-            list.add(new String[] { cmd_key, cmd_name, src_type, src_name, src_desc });
+            list.add(new Pair(new String[] { cmd_key, cmd_name, src_type, src_name, src_desc }, configurable));
```

Comment:
People usually use `Pair.of()` to make pairs.

Reply:
fixed - makes sense. I never know which of these little things incorporates which of the little toys.